runtime/src/iree-amd-aie/aie_runtime/test/DeviceModelTest.cpp

```diff
@@ -413,8 +413,8 @@ TEST_P(AMDAIENPUDeviceModelParameterizedSixTupleNPU4ColTestFixture,
     auto destWireb = STRM_SW_PORT_TYPE_TO_WIRE_BUNDLE(destSw);
     auto deviceModelIsLegal = deviceModel.isLegalMemtileConnection(
         c, r, srcSw, srcChan, destSw, dstChan);
-    auto targetModelIsLegal = targetModel.isLegalMemtileConnection(
-        srcWireB, srcChan, destWireb, dstChan);
+    auto targetModelIsLegal = targetModel.isLegalTileConnection(
+        c, r, srcWireB, srcChan, destWireb, dstChan);
```

Comment:
Ah this isn't gonna work - we're not using Air target model we're using [Device model](https://github.com/nod-ai/iree-amd-aie/blob/5a7ebc899a0aa26706ee2e52f85bda5609948bb7/runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h#L227). Do your changes depend on the recent updates to AIETargetModel?

Reply:
Yeah this recent PR is introducing changes to AIETargetModel: https://github.com/erwei-xilinx/mlir-aie/commit/c04ffea6dc2430b65557feca3ff6a7f55a2e64b5